### PR TITLE
Allow using modern versions of ActiveMerchant

### PR DESCRIPTION
**Pull request by mroach**
1.49.0 is over a year old. Missing many new gateways, fixes, and improvements

### CODE REVIEW

File: core/spree_core.gemspec [hunk 1]
```diff
@@ -19,7 +19,7 @@ Gem::Specification.new do |s|
   s.files        = `git ls-files`.split("\n")
   s.require_path = 'lib'
 
-  s.add_dependency 'activemerchant', '~> 1.49.0'
+  s.add_dependency 'activemerchant', '>= 1.49', '< 2.0'
```
Comment (krzysiek1507) [inline core/spree_core.gemspec]: You can drop `.0` and leave `~> 1.49`

Comment (mroach) [inline core/spree_core.gemspec]: Righto. Updated.

**Comment by mroach** [comment]
I merged 3-1-stable into my branch to keep it up to date and now the CircleCI tests pass. Though looking at the failure log from the previous run, it seems like there was some kind of networking error with CircleCI itself that had nothing to do with the spree code.

**Comment by mroach** [comment]
This PR is no longer relevant. I hadn't noticed @c1b26b077cbef1c44ee095a33e580200d1efc42b

**Pull request closed without merging**